changelogs/fragments/59272-support-for-entrust-provider-in-openssl_certificate_module.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - openssl_certificate - Add support for a new provider ''entrust'' (https://github.com/ansible/ansible/pull/59272).
```

```suggestion
  - openssl_certificate - Add support for a new provider ``entrust`` (https://github.com/ansible/ansible/pull/59272).
```
It should be backticks, for [ReST](https://en.wikipedia.org/wiki/ReStructuredText) formatting.